### ISSUE TITLE
sharpen-simple-navigation needs to support a button (not link) for login

### ISSUE THREAD

**Issue by prschmid** [coggen/kezuri#33]
To get to the Auth0 login page, we need to make a POST request and not a GET request. 

Fore reference, here is a snippet of code that successfully gets us to the Auth0 login page and what we need to be able to support.
```
<%= button_to t('buttons.log_in'),
                  '/auth/auth0',
                  id: 'login-button',
                  data: { turbo: "false" },
                  class: 'link',
                  method: :post %>
```
Maybe instead of making the link data a parameter to the component, can we pass it something instead in case we need to change how this works?

**Comment by jabbett** [comment]
I am creating a general purpose slot for custom navigation in this component. I will also create a new class for styling buttons as simple links. Example:

```html
<sharpen-simple-navigation background="sunrise" brand-href="https://sharpen.com">
  <div slot="custom-nav">
    <button class="sharpen-link">Log In</button>
  </div>
</sharpen-simple-navigation>
```

**Comment by jabbett** [comment]
![image](https://github.com/coggen/kezuri/assets/923971/cb1e4f25-b880-4b69-b81a-1f3fd4a1796f)

**Comment by jabbett** [comment]
Released in 0.11.0
